### PR TITLE
feat(plugins): Load plugins from "plugins" folder

### PR DESCRIPTION
**Pull request by 0x-FADED**
load plugins form "Plugins" folder instead of loading form root peacock installation directory

### CODE REVIEW

**Review by AnthonyFuller — CHANGES_REQUESTED** [review]

File: components/controller.ts [hunk 1]
```diff
@@ -869,19 +869,20 @@ export class Controller {
...
     private async _loadPlugins(): Promise<void> {
-        const entries = (await readdir(process.cwd())).filter(
+        const entries = (await readdir(join(process.cwd(), "plugins"))).filter(
             (n) => isPlugin(n, "js") || isPlugin(n, "cjs"),
         )
 
         for (const plugin of entries) {
-            const src = (await readFile(plugin)).toString()
+            const sourceFile = join(process.cwd(), "plugins", plugin),
+            src = (await readFile(sourceFile)).toString();
```
Comment (RDIL) [inline components/controller.ts:883]: Style: please use 2 const declarations instead of comma-trailing them.  Also, run `yarn prettier`.

**Comment by 0x-FADED** [comment]
squashed commits ran prettier, and a way to load form plugins folder with backwards compatibility

**Comment by RDIL** [comment]
Not sure how you ran prettier, but that's not our code style.

**Comment by AnthonyFuller** [comment]
And you _still_ haven't resolved the typo in your comment.

**Comment by 0x-FADED** [comment]
re-ran prettier but this time project is failing to pass checks for some reason

**Pull request merged**